### DISCUSSION TITLE
Rating the rater

### DISCUSSION THREAD

**Discussion by alserembani94** [farhan-helmy/ratemysurau#5] [Ideas]
The idea is, kita nak improve rating system. The inspiration datang daripada Google Places, where user punya score is higher bila dia banyak rate surau. So, this boleh leverage influencer untuk libat diri untuk rating as well.

Somehow macam gamification sikit la. So, bila score dia tinggi, kita boleh build other feature in future, contohnya affiliation dengan surau, external voucher system (ni just bagi gambaran apa kita boleh buat bila ada feature ni).

**Comment by farhan-helmy** [comment]
ok ni tapi nak kena implement auth

**Reply by alserembani94** [reply]
Hmmm betul jugak 😅

Boleh put later in stage la kot.

**Reply by alserembani94** [reply]
Chotto matte!

Aku tak tau kalau suggestion aku ni okay ke tak, tapi boleh try tengok Magic Auth.

Dia auth guna web3, siap social login semua. Aku tau web3 tu macam overkill, tapi kalau ko tak nak setup auth, aku rasa Magic Auth tu antara yang plaing simple, sebab dia pakai DID (Decentealised ID).

Probably good use case kalau ko nak consider.
